src/dbghelp.rs

```diff
@@ -114,9 +114,8 @@ macro_rules! dbghelp {
             })*
 
             pub fn dbghelp(&self) -> *mut Dbghelp {
-                unsafe {
-                    ptr::addr_of_mut!(DBGHELP)
-                }
+                #[allow(unused_unsafe)]
+                unsafe { ptr::addr_of_mut!(DBGHELP) }
```

Comment:
Maybe we can spell it `ptr::addr_of!(DBGHELP).cast_mut()`? Just to emphasise that we're actually getting a const pointer but casting it to `*mut` for *reasons*.